test/zero_dim_tensor_elimination.cpp

```diff
@@ -55,26 +55,6 @@ TEST(zero_dim_tensor_elimination, zero_sum)
     ASSERT_EQ(count_ops_of_type<op::Sum>(f), 0);
 }
 
-TEST(zero_dim_tensor_elimination, zero_const_conv)
-{
-    Shape zero_shape{0};
-    auto A = std::make_shared<op::Parameter>(element::f32, Shape{1, 1, 0});
-    auto weights = std::make_shared<op::Parameter>(element::f32, Shape{1, 1, 4});
```

Comment:
Attn: @Krovatkin. This is actually invalid (weights are bigger than the data batch along spatial dims, and dims has a spatial dimension of zero, neither of which are "supposed to be" allowed), so this test is relying on a bug in validation. We could relax the spec of the convolution op to allow dim-zero axes on both data and filters.